### PR TITLE
Azure case study

### PR DESCRIPTION
**Pull request by ArneBHuset**
Case study about Microsoft Azure ready for review

### CODE REVIEW

File: src/content/docs/webdev-hosting/azure/azure.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+- 2017: Axure Comsos DB, a globally distributed, multi-model database service was announced. 
+- 2018: Azure kubernetes services (AKS) was announced, offering managed Kubernetes on Azure. Confidential computing was made available, adding new data security technologies. 
+- 2019: Microsoft announced Azure Blockchain Service, making it simpler to deploy secure blockchain networks
+- 2020: New AI-driven data analytics platform called Azure Synapse Analaytics was introduced. 
+- 2021: Microsoft Mesh was announced, offering mixed realtiy capabilities integrated with Azure. 
+- 2022: Azure OpenAI was launched. Integrating AI models like GPT-3 into its cloud offering
+- 2023: More advanced AI and machine learning tools are made available. Increased efforts in Azures sustainability efforts, including tools to measure and manage carbon emissions. 
+- 2024: Ongoing plans to move Azure's global infrastructure towards, and further integrating it with quantom computing technologies. 
+
+## Bacground
```
Comment (sulenchy) [inline src/content/docs/webdev-hosting/azure/azure.md:28]: There is a typo here

**Review by sulenchy — CHANGES_REQUESTED** [review]
@ArneBHuset 
Thanks for the effort you put into raising this PR. Unfortunately, this is a duplicate. There are already three case studies in the codebase, and according to the brief of the CA, you are expected to either consolidate the existing ones into one or delve deeper into one of the features of Azure in your research.

Please fix the issue and get back to me within the next 24 hours.

**Comment by ArneBHuset** [comment]
Hi,

I asked Oliver if i could write about Azure since i wanted to learn more about the basis of it and got a thumbs up.

Regards
Arne

Sendt fra Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by sulenchy** [comment]
@ArneBHuset That is understandable. The issue with your PR is that it duplicates an existing case study and there is a section in the brief that advises you on what to do if someone has written about your topic. See

```
If there is an existing case study discussing your chosen product, consider these contribution options:

Create a video companion to an existing written case study
Create a text companion to an existing video case study
Pick a particular feature, offering or aspect of this product to deep dive into
Consolidate duplicate case studies into a single case study on that subject
```

To pass the CA, you need to comply with the requirements stated in the brief.

**Comment by ArneBHuset** [comment]
I asked if i could write about Azure from the beginning instead of building on top, seeing as i have to grasp the basics to be able to comprehend anything else. I am very interested in Azure, and this was the appropriate way to learn about it for me, i would see no purpose in skipping all the basics just to continue with a more detailed case study. I saw the requirements, therefore i asked Oliver if it would be okay and i got a thumbs up

-Arne

Sendt fra Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Fermain** [comment]
I recall discussing it, bringing up the fact that many have already chosen this and suggesting focusing on one particular azure product in more detail. @ArneBHuset do you think that's a fair recollection of our chat?

**Comment by sulenchy** [comment]
Closed this because

**Comment by sulenchy** [comment]
I closed this because the PR duplicated an existing Azure case study

**Pull request closed without merging**